bundles/ilg.gnumcueclipse.debug.gdbjtag.jumper/src/ilg/gnumcueclipse/debug/gdbjtag/jumper/preferences/DefaultPreferences.java

```diff
@@ -74,7 +72,7 @@
 	// ------------------------------------------------------------------------
 
 	// HKCU & HKLM LOCAL_MACHINE
-	private static final String REG_SUBKEY = "\\GNU ARM Eclipse\\QEMU";
+	private static final String REG_SUBKEY = "\\GNU ARM Eclipse\\Jumper";
```

Comment:
I suggest you completely remove support for the Windows registry, I'm moving away from Windows setups towards portable archives and binary xPacks.

Reply:
removed it altogether as it wasn't used